package.json

```diff
@@ -91,14 +91,13 @@
   "dependencies": {
     "algoliasearch": "^3.29.0",
     "algoliasearch-helper": "^2.26.1",
+    "angular-instantsearch": "3.0.0-beta.0",
```

Comment:
This does not seems to be correct, you added `angular-instantsearch` as a dependency of itself :innocent:

Reply:
😅